src/main/resources/assets/iris/lang/ru_ru.json

```
@@ -31,7 +34,7 @@
   "options.iris.shaderPackSettings": "Настройки набора шейдеров",
   "options.iris.shaderPackList": "Список наборов шейдеров",
   "options.iris.refreshShaderPacks": "Обновить список шейдеров",
-  "options.iris.shaderPackSelection": "Наборы шейдеров...",
+  "options.iris.shaderPackSelection": "Наборы шейдеров…",
```

```suggestion
  "options.iris.shaderPackSelection": "Наборы шейдеров...",
```
Не стоит менять это написание, так как это приведёт к нарушению согласованности с текущей локализацией игры. Дело в том, что символ многоточия имеет бо́льшую ширину, нежели 3 отдельных точки, что используется в русской локализации для сокращения занимаемого текстом пространства.
В оригинальных строках этот символ тоже со временем перестали использовать.

---

Пойду накажу автоформатирование кода за это.